src/highlight/__tests__/HighlightList-test.tsx

```diff
@@ -49,4 +51,71 @@ describe('HighlightList', () => {
             expect(useOutsideEvent).toHaveBeenCalledTimes(2);
         });
     });
+
+    describe('filterHighlight()', () => {
```

Comment:
Can we test this via render, instead? Currently, it only tests an array with a length of 1.